components/MadeWithPrezly/MadeWithPrezly.tsx

```diff
@@ -0,0 +1,19 @@
+import { LogoPrezly } from 'icons';
+
+import styles from './MadeWithPrezly.module.css';
+
+export function MadeWithPrezly() {
+    return (
+        <a
+            // TODO: replace with new landing page URL
+            href="https://prezly.com"
```

Comment:
Shouldn't this be `https://prez.ly/storytelling-platform`?

Reply:
Right!